certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
 ## 4.1 Certificate Application
 
 ### 4.1.1 Who can submit a certificate application
-In accordance with Section 5.5.2, the CA SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. The CA SHALL use this information to identify subsequent suspicious certificate requests.
+In accordance with Section 5.5.2, all CAs SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. All CA SHALL use this information to identify subsequent suspicious certificate requests.
+
+For the Root and Subordinate CAs:
+- An application for a CA certificate shall be submitted by an authorized representative of the applicant CA.
+
+For end entity certificates:
+- A certificate application shall be submitted to the CA by the Subscriber, an authorized organization representative, or an RA on behalf of the Subscriber. 
+
 
 ### 4.1.2 Enrollment process and responsibilities
-Prior to the issuance of a Certificate, the CA SHALL obtain the following documentation from the Applicant:
+For the Root and Subordinate CAs:
+- The Policy Authority is responsible for approving or denying requests for CA certificate issuances.
+
+For all CAs, prior to the issuance of any Certificate, the CA SHALL obtain the following documentation from the Applicant:
 
 1. A certificate request, which may be electronic; and
 2. An executed Subscriber Agreement or Terms of Use, which may be electronic.
 
-The CA SHOULD obtain any additional documentation the CA determines necessary to meet these Requirements.
+The certificate request SHALL contain a request from, or on behalf of, the Applicant for the issuance of a Certificate, and a certification by, or on behalf of, the Applicant that all of the information contained therein is correct.
```

Comment:
Got it, so for DV it sounds like it's basically a NOOP (since there is no other information validated in the cert). I can understand how this works as boilerplate language.